routers/repo/view.go

```diff
@@ -333,7 +333,14 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 	// Show latest commit info of repository in table header,
 	// or of directory if not in root directory.
 	ctx.Data["LatestCommit"] = latestCommit
-	ctx.Data["LatestCommitVerification"] = models.ParseCommitWithSignature(latestCommit)
+	verification := models.ParseCommitWithSignature(latestCommit)
+
```

Comment:
didn't you have to call CalculateTrustStatus and the set `ctx.Data["LatestCommitVerification"] = verification` ?

Reply:
ok it should work because you only save the pointer to Data but it would be more cleare in the code if you sitch this :)

Reply:
@zeripath ok it should work because you only save the pointer to Data but it would be more cleare in the code if you sitch this :)

Reply:
fine.